### ISSUE TITLE
urllib.error.HTTPError: HTTP Error 404: Not Found

### ISSUE THREAD

**Issue by bkavlak** [giswqs/whitebox-python#23]
* whitebox version: 1.4.0
* Python version: 3.6
* Operating System: Linux 18.04 LTS

### Description

Hey, I'm getting error in the Docker geemap container when import whitebox for the first time. 
![error](https://user-images.githubusercontent.com/36866123/115448597-880be200-a222-11eb-9fcf-c6cadab8238b.jpg)

It is not a problem if I type again an import. I couldn't understand why this throws an error. Here is a related issue:
https://stackoverflow.com/questions/42441211/python-urllib-error-httperror-http-error-404-not-found

I guess this is not an important error and related to network issues. I wanted to add in case someone sees the error message.

### What I Did

```
docker run -it --name geemap_latest -p 8888:8888 -p 6006:6006 -v C:/Users/batuh/Desktop:/geemap/data bkavlak/geemap:latest bash
import whitebox
```

**Comment by giswqs** [comment]
whitebox needs to download the WhiteboxTools binary for the first time use. I am not sure why it would throw the urllib error. It works fine on my local computer.

**Comment by bkavlak** [comment]
Yes, it seems we can ignore it.

**Comment by giswqs** [comment]
I just ran into the same issue. It seems the testdata were no longer available from the WhiteboxTools repo. I will update the URLs shortly to fix this. 

![image](https://user-images.githubusercontent.com/5016453/115483221-5e50bc00-a21e-11eb-8800-36025590eb03.png)

**Comment by giswqs** [comment]
This issue has been fixed in whitebox v1.4.1
